specs/posix_glob_spec.yaml

```diff
@@ -0,0 +1,81 @@
+specify posix.glob:
+
+- before:
+    glob = require "posix.glob"
+
+- describe glob.GLOB_MARK:
+  - before:
+      MARK = glob.GLOB_MARK
+
+  - it is an number:
+      expect (type(MARK)).to_be "number"
+
+- describe glob:
+  - before:
+      glob, MARK = glob.glob, glob.GLOB_MARK
+      chdir, mkdir, mkdtemp = posix.chdir, posix.mkdir, posix.mkdtemp
+
+  - context with bad arguments:
+      badargs.diagnose (glob, "(?string, int)")
+
+  - it returns nil and an error message if there are no matches:
+      dir = mkdtemp (template)
+      chdir (dir)
+      globlist, errmsg = glob("foo", 0)
+      expect (type(errmsg)).to_be "string"
+      expect (errmsg).to_be("foo:" .. " No such file or directory")
```

Comment:
By the way, this is odd. I had to glue together these two strings because the test runner wasn't happy with a colon followed by a space, even inside a string. I'm guessing this is an issue with YAML parsing, but I'm not sure.